PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-appservice/src/main/java/com/microsoft/azure/toolkit/intellij/legacy/function/runner/localrun/FunctionRunConfiguration.java

```diff
@@ -225,7 +233,7 @@ public void validate() throws ConfigurationException {
         if (!func.exists() || !func.isFile() || !func.getName().contains("func")) {
             throw new ConfigurationException(message("function.run.validate.invalidFuncPath"));
         }
-        if (getFuncPort() <= 0 || getFuncPort() >= 65535) {
+        if (!isAutoPort() && (getFuncPort() <= 0 || getFuncPort() >= 65535)) {
```

Comment:
any port (no matter `auto` or not) should also be restricted in `(0, 65536)`,

Reply:
assign it to local variable if `getFuncPort` is heavy.

Reply:
`getFuncPort` is just a get method without any calculation

Reply:
> any port (no matter `auto` or not) should be restricted in `(0, 65536)`,

In case users set a wrong port but switch to auto port, if we validate the port in any cases, users need to switch back to manual port mode and correct the port, then switch back to auto, which may be a bit annoying